TAF_EasyRest/TestFramework/PageComponents/NavigationMenuPageComponents/SignInSignUpHeaderButtons.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TestFramework.PageComponents.NavigationMenuComponents
+{
+    public class SignInSignUpHeaderButtons
```

Comment:
Add PageComponents in the end of the name.

Reply:
added